### DISCUSSION TITLE
该如何实现对超长日志进行截取

### DISCUSSION THREAD

**Discussion by XTUxiongda** [alibaba/ilogtail#1222] [Help] [answered]
就是打了一条超长的日志，源头是文件，输出端是kafka
我想在ilogtail采了，超过一定长度，比如3000，先截取了再丢到kafka，在官网上没找到对应的函数，目前支持吗？

**Comment by 7y-9** [comment]
可以自己实现一个processor的，把log.Contents 里面的content的value截取就好。

**Comment by linrunqi08** [comment, marked as answer]
@XTUxiongda  这个目前没有插件进行支持，有需求欢迎一起参与开发
